pom.xml

```diff
@@ -50,29 +50,38 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.8.1</version>
         <configuration>
-          <compilerId>javac-with-errorprone</compilerId>
           <source>9</source>
           <target>9</target>
           <encoding>${project.build.sourceEncoding}</encoding>
           <forceJavacCompilerUse>true</forceJavacCompilerUse>
           <showWarnings>true</showWarnings>
           <compilerArgs>
-            <!-- workaround to avoid the bug in errorprone https://github.com/google/error-prone/issues/780#issuecomment-357952383 -->
-            <arg>-Xep:ParameterName:OFF</arg>
+            <arg>-XDcompilePolicy=simple</arg>
+            <arg>
+              -Xplugin:ErrorProne
+              <!-- workaround to avoid the bug in errorprone https://github.com/google/error-prone/issues/780#issuecomment-357952383 -->
+              -Xep:ParameterName:OFF
+            </arg>
+            <arg>--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED</arg>
+            <arg>--add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED</arg>
+            <arg>--add-exports=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED</arg>
+            <arg>--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED</arg>
+            <arg>--add-exports=jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED</arg>
+            <arg>--add-exports=jdk.compiler/com.sun.tools.javac.main=ALL-UNNAMED</arg>
+            <arg>--add-exports=jdk.compiler/com.sun.tools.javac.model=ALL-UNNAMED</arg>
+            <arg>--add-exports=jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED</arg>
+            <arg>--add-exports=jdk.compiler/com.sun.tools.javac.processing=ALL-UNNAMED</arg>
+            <arg>--add-exports=jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED</arg>
+            <arg>--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED</arg>
```

Comment:
Looks like the plugin isn't configured to fork, so IIUC most/all of these arguments can be omitted thanks to the flags in `.mvn/jvm.config`.

Reply:
Aren't those more about the APIs that errorprone-slf4j uses, and .mvn/jvm.config about those required by ErrorProne to compile errorprone-slf4j sources?

Reply:
Right, yes: we need both the compile-time flags and the runtime flags here.

Reply:
Sorry, I mixed stuff up (and forgot that within our own Error Prone-based project we don't have this configuration because instead there's a dependency on `com.google.errorprone:javac`).

NB: `--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED` is listed twice :)

Reply:
I removed the dupe, thanks :)